examples/platforms/da15000/radio.c

```diff
@@ -0,0 +1,551 @@
+{
+    (void)aInstance;
+    return s_promiscuous;
+}
+
+ThreadError otPlatRadioHandleTransmitDone(bool *rxPending)
+{
+    ThreadError error = kThreadError_None;
+    VerifyOrExit(s_state == kStateTransmit, error = kThreadError_InvalidState);
+
+    s_state = kStateReceive;
+
+    if (rxPending != NULL)
+    {
+        *rxPending = s_data_pending;
+    }
+
+exit:
+    return error;
+}
+
+
+
+
+void FTDF_sendFrameTransparentConfirm(void         *handle,
+                                      FTDF_Bitmap32 status)
+{
+    (void)handle;
+    (void)status;
+    volatile uint32_t *txStatus = FTDF_GET_REG_ADDR_INDEXED(RETENTION_RAM_TX_RETURN_STATUS_1, FTDF_TX_DATA_BUFFER);
+
+    SendFrameDone = true;
+
+#if DEBUG_LOG_ENABLE
+    hw_uart_write_buffer(HW_UART1, "\nTD", 3);
+#endif
+}
+
+
+
+void da15000RadioProcess(otInstance *aInstance)
+{
+    bool rxPending;
+    uint8_t value = 0;
+
+    if (SendFrameDone)
+    {
+        FTDF_SET_FIELD(ON_OFF_REGMAP_RXENABLE, 1);
+        otPlatRadioHandleTransmitDone(&rxPending);
+        otPlatRadioTransmitDone(aInstance, &s_transmit_frame, false, kThreadError_None);
+        SendFrameDone = false;
+    }
+
+    if (SED)
+    {
+        if ((otPlatAlarmGetNow() - sleep_const_delay) > 100)
+        {
+            FTDF_setValue(FTDF_PIB_RX_ON_WHEN_IDLE, &value);
+            s_state = kStateSleep;
+            SED = false;
+        }
+
+    }
+}
+
+
+
+void FTDF_rcvFrameTransparent(FTDF_DataLength frameLength,
+                              FTDF_Octet     *frame,
+                              FTDF_Bitmap32   status)
+{
+    (void)status;
+    RadioPacket otRadioPacket;
+    //  int8_t        value;
+    //  value  = -20;
+
+    otRadioPacket.mPower = -20;           //FIX THIS -actual measurements shall be placed here
+    otRadioPacket.mLqi = kPhyNoLqi;
+
+    if (s_receive_frame.mChannel == 0)
+    {
+        s_receive_frame.mChannel = 11;    //FIX this - dirty fix for channel handling.
+    }
+
+
+    otRadioPacket.mChannel =  s_receive_frame.mChannel;
+    otRadioPacket.mLength = frameLength;
+    otRadioPacket.mPsdu = frame;
+
+    otPlatRadioReceiveDone(First_Instace, &otRadioPacket, kThreadError_None);
+
+    if (s_state != kStateDisabled)
+    {
+        s_state = kStateReceive;
+    }
+
+}
+
+void otPlatRadioGetIeeeEui64(otInstance *aInstance, uint8_t *aIeeeEui64)
+{
+    (void)aInstance;
+    aIeeeEui64[0] = 0x18;
+    aIeeeEui64[1] = 0xb4;
+    aIeeeEui64[2] = 0x30;
+    aIeeeEui64[3] = 0x00;
+    aIeeeEui64[4] = (NODE_ID >> 24) & 0xff;
+    aIeeeEui64[5] = (NODE_ID >> 16) & 0xff;
+    aIeeeEui64[6] = (NODE_ID >> 8) & 0xff;
+    aIeeeEui64[7] = NODE_ID & 0xff;
+}
+
+ThreadError otPlatRadioEnergyScan(otInstance *aInstance, uint8_t aScanChannel, uint16_t aScanDuration)
+{
+    (void)aInstance;
+    (void)aScanChannel;
+    (void)aScanDuration;
+    return kThreadError_NotImplemented;
+}
+
+void otPlatRadioEnableSrcMatch(otInstance *aInstance, bool aEnable)
+{
+
+    (void)aInstance;
```

Comment:
If you going to have `(void)aInstance`, then you should probably also have `(void)aEnable`